### DISCUSSION TITLE
AMD GPU on Windows Support

### DISCUSSION THREAD

**Discussion by Nealtron** [mathoudebine/turing-smart-screen-python#47] [Q&A]
Is there a Feature-request or issue I can follow to be notified when AMD GPU support on Windows is available? I'd love to leave the sketchy "USBMonitor.exe" app behind ASAP, but I watch GPU Utilization heavily.

**Comment by mathoudebine** [comment]
Hi, good news for you: the AMD GPUs are supported on Windows!
You can use the latest release available (2.0.3)
The library used is [pyadl](https://github.com/nicolargo/pyadl)
The GPU memory usage is still unsupported, but you can have the GPU temperature & usage (load) in %

**Reply by Nealtron** [reply]
Interesting, as I'm not currently able to get and GPU output. I'll open an issue and attach the output.
